src/OpenTelemetry/Metrics/Builder/MeterProviderBuilderExtensions.cs

```diff
@@ -314,6 +317,40 @@ public static MeterProvider Build(this MeterProviderBuilder meterProviderBuilder
         throw new NotSupportedException($"Build is not supported on '{meterProviderBuilder?.GetType().FullName ?? "null"}' instances.");
     }
 
+#if EXPOSE_EXPERIMENTAL_FEATURES
+    /// <summary>
+    /// Sets a positive integer value defining the maximum number of
+    /// data points allowed for metrics managed by the MeterProvider.
```

Comment:
```suggestion
    /// metric points allowed for metrics managed by the MeterProvider.
```

Using metric points here is more accurate?
In the data-model, metric points denotes a set of data points, each containing an independent set of attribute name-value pairs.